README.md

```diff
@@ -188,7 +188,18 @@ Type: `Boolean`
 Default: false
 
 Pass this flag if you don't care about full compliance with Internet Explorer 6-8 quirks.
-
+  
+#### quote_style
```

Comment:
Finally, `quoteStyle` seems consistent with the scheme that's already being used, i.e. camelcase.